### PR TITLE
Added larger list from www.bannedwordlist.com/lists/swearWords.csv

### PR DESCRIPTION
**Pull request by tliu22**
Added larger list from www.bannedwordlist.com/lists/swearWords.csv, especially in light of quick-validation and recommendation system.

### CODE REVIEW

**Comment by sahilhasan** [comment]
I would personally say we should remove lmao, lmfao and wtf. None of those are 'bad' enough to warrant not allowing imo

File: ocflib/account/creation.py [hunk 1]
```diff
@@ -29,7 +29,12 @@
 
 
 _KNOWN_UID = 50500
-BAD_WORDS = frozenset(('fuck', 'shit', 'cunt', 'bitch', 'dick'))
+BAD_WORDS = frozenset(('anal', 'anus', 'arse', 'ass', 'ballsack', 'balls', 'bastard', 'bitch', 'biatch', 'bloody',
+'blowjob', 'bollock', 'bollok', 'boner', 'boob', 'bugger', 'bum', 'butt', 'buttplug', 'clitoris', 'cock', 'coon',
+'crap', 'cunt', 'damn', 'dick', 'dildo', 'dyke', 'fag', 'feck', 'fellate', 'fellatio', 'felching', 'fuck',
+'fudgepacker', 'flange', 'Goddamn', 'hell', 'homo', 'jerk', 'jizz', 'kike', 'knobend', 'labia', 'lmao', 'lmfao', 'muff',
```
Comment (matthew-mcallister) [inline ocflib/account/creation.py]: Why have "Goddamn" if "damn" already appears? Also that "G" shouldn't be capitalized.

File: ocflib/account/creation.py [hunk 2]
```diff
@@ -327,13 +332,13 @@ def validate_username(username, realname):
 
     if similarity_heuristic(realname, username) > SIMILARITY_THRESHOLD:
         raise ValidationWarning(
-            'Username {} not based on real name {}'.format(username, realname))
+            'Username {} not based on real name {}.'.format(username, realname))
```
Comment (matthew-mcallister) [inline ocflib/account/creation.py]: Isn't this https://github.com/ocf/ocflib/pull/110? These changes shouldn't go here I don't think.

**Comment by matthew-mcallister** [comment]
Alright, this list is not bad. I think some duplicate words need removing (see my comment for an example). Also, the indentation should be fixed to look like
```python
xyz = frozenset({
    'foo', 'bar', 'baz', ...
})
```

I agree with @sahilhasan's comments. In particular, `wtf` are valid initials and `lmao` could be based on a real name (i.e. last name Mao).

**Comment by tliu22** [comment]
Oh, I might've missed some duplicate words. Are there any more?

And I thought 'wtf' and 'lmao' were already removed?

**Comment by sahilhasan** [comment]
@tliu22 RIP I looked at the CSV (and I remembered from the previous list). Thats my fault. Sry

**Comment by matthew-mcallister** [comment]
@tliu22 "balls" and "ballsack"

**Comment by kpengboy** [comment]
Late to the game, but we should also expand the test cases to cover these new bad words.

**Comment by kpengboy** [comment]
And we need to get the Scunthorpe project done soon...

**Pull request merged**